src/Interfaces/AccessDefinitionOwnerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace HalloVerden\Security\Interfaces;
+
+/**
+ * Interface AccessDefinitionOwnerInterface
+ *
+ * @package HalloVerden\Security\Interfaces
+ */
+interface AccessDefinitionOwnerInterface {
+
+  /**
+   * @param AccessDefinitionOwnerInterface $owner
+   *
+   * @return bool
+   */
+  public function equalAccessDefinitionOwner(AccessDefinitionOwnerInterface $owner): bool;
```

Comment:
Would rename method to 'isEqualTo'

Reply:
I did not want to "collide" with any common method names. 'isEqualTo' might be to generic?

Reply:
@tomme87 Fair point. How about 'isEqualToAccessDefinitionOwner'?